include/effects/matrix/PatternPongClock.h

```diff
@@ -309,7 +309,7 @@ class PatternPongClock : public LEDStripEffect
         LEDMatrixGFX::backgroundLayer.fillRectangle(BAT1_X - 1, bat1_y, BAT1_X, bat1_y + BAT_HEIGHT, rgb24(255,255,255));
         //      }
 
-        // move bat 2 towards target (dont go any further or bat will move off screen)
+        // move bat 2 towards target (dont go any further or bat will move off-screen)

```

Comment:
Same comment as before. Also, "dont" does need an apostrophe.

```suggestion
        // move bat 2 towards target (don't go any further or bat will move off screen)
```